ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -140,6 +127,17 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
+    <Reference Include="Ionic.Zip">
```

Comment:
Looks like this shuffle isn't needed? Perhaps you could move these back manually to avoid code churn // difficulty merging for other users.